src/js/actions/AppsActions.js

```diff
@@ -170,6 +170,12 @@ var AppsActions = {
         });
       });
   },
+  pronounceAppStatusesCount: function (appsStatusesCount) {
```

Comment:
Sorry I'm really getting on my own nerves with these naming comments... but "pronounce" is such a weird word to pick here. "announce"? "emit"? "resolve"? "declare"?